libgnucash/engine/Account.c

```diff
@@ -4444,14 +4444,15 @@ xaccAccountGetReconcilePostponeBalance (const Account *acc,
     g_return_val_if_fail(GNC_IS_ACCOUNT(acc), FALSE);
     qof_instance_get_kvp (QOF_INSTANCE(acc),
                           "reconcile-info/postpone/balance", &v);
-    if (G_VALUE_HOLDS_INT64 (&v))
+    if (G_VALUE_HOLDS_INT64 (&v)) {
         bal = *(gnc_numeric*)g_value_get_boxed (&v);
 
-    if (bal.denom)
-    {
-        if (balance)
-            *balance = bal;
-        return TRUE;
+        if (bal.denom)
```

Comment:
Yes it could be rewritten like that and personally I would even prefer that more as I think it actually increases the readability rather than decreasing it. I understand @jmarino didn't do this though to stay as close as possible to the existing code.

I would even go one step further and also change the first test ```if (G_VALUE_HOLDS_INT64(&v)) { ... }``` to ```if (!G_VALUE_HOLDS_INT64(&v)) return FALSE``` to reduce another level of indentation.

In addition I'd remove the curly braces of your two if clauses, considering the code to execute on true is only one single statement. (And your last line is missing a semi-colon ;) )

Feel free to submit your improvements as a separate PR (and feel free to incorporate my amendments) and I'll happily merge them in. Thanks!

P.S. You may want to go through the whole file, or at least the changes as found in the commit I mentioned in another comment and make similar adjustments in several other places.

Reply:
I have never contributed to open source before - let alone a c/c++ project. This sounds like an interesting way to get familiar with some parts of the codebase and also contribute back.

I'm compiling gnucash on my laptop right now and reading articles on the wiki. I've also joined IRC.

Thanks for the encouragement :) I'll see what I can do.